src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -609,6 +610,59 @@ public void AssemblyResolve_RequestingAssembly()
         }
 
         [Fact]
+        [ActiveIssue("https://github.com/dotnet/corefx/issues/22555", TargetFrameworkMonikers.Uap)]
```

Comment:
I just wanted to unblock the CoreCLR fix, I'll be looking at the that issue for uap next